### ISSUE TITLE
Add true persistence for stored data

### ISSUE THREAD

**Issue by jagoe** [RauchF/BabelBot#20]
Currently, BabelBot stores data in memory and will lose it after a restart.
Instead, BabelBot should store data persistently.

- There should be a storage default that doesn't rely on external databases.
- BabelBot administrators should be able to point it to an existing database instance.
- The Helm chart should reflect those options and provide a default that doesn't need any configuration.

**Comment by RauchF** [comment]
Good point. I've added this issue to the Multi-Node Support milestone as I believe it's more or less essential for that too :)
